src/components/MenuItem.tsx

```diff
@@ -1,96 +1,77 @@
 import React, { ReactElement } from 'react'
-import { View, Text, Pressable, Image, StyleSheet } from 'react-native'
+import { View, Pressable} from 'react-native'
 import { Arrow } from '../../assets/images'
 import { COLORS } from '../constants/colors'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+import styled from 'styled-components/native'
 
-export const styles = StyleSheet.create({
-  wrapper: { paddingHorizontal: 16 },
-  container: {
-    justifyContent: 'space-between',
-    alignSelf: 'center',
-    paddingVertical: 17,
-    paddingRight: 8,
-    paddingLeft: 16,
-    marginBottom: 8,
-    flexDirection: 'row',
-    alignItems: 'center',
-    width: '100%',
-    backgroundColor: COLORS.white,
-    borderColor: COLORS.lunesBlackUltralight,
-    borderWidth: 1,
-    borderStyle: 'solid',
-    borderRadius: 2
-  },
-  clickedContainer: {
-    marginHorizontal: 16,
-    justifyContent: 'space-between',
-    alignSelf: 'center',
-    paddingVertical: 17,
-    paddingRight: 8,
-    paddingLeft: 16,
-    marginBottom: 8,
-    flexDirection: 'row',
-    alignItems: 'center',
-    width: '100%',
-    backgroundColor: COLORS.lunesBlack,
-    borderColor: COLORS.white
-  },
-  itemTitle: {
-    fontSize: wp('5%'),
-    fontWeight: '600',
-    letterSpacing: 0.11,
-    marginBottom: 2,
-    color: COLORS.lunesGreyDark,
-    fontFamily: 'SourceSansPro-SemiBold'
-  },
-  clickedItemTitle: {
-    fontSize: wp('5%'),
-    fontWeight: '600',
-    letterSpacing: 0.11,
-    marginBottom: 2,
-    color: COLORS.white,
-    fontFamily: 'SourceSansPro-SemiBold'
-  },
-  icon: {
-    justifyContent: 'center',
-    marginRight: 10,
-    width: wp('7%'),
-    height: wp('7%')
-  },
-  left: { flexDirection: 'row', alignItems: 'center' }
-})
+const Wrapper = styled.View`
+   padding-left: 16;
+   padding-right: 16;
+`;
+const ItemStyle = styled(Pressable)`
+    justify-content: space-between;
+    align-self: center;
+    padding-top: 17;
+    padding-bottom: 17;
+    padding-right: 8;
+    padding-left: 16;
+    margin-bottom: 8;
+    flex-direction: row;
+    align-items: center;
+    width: 100%;
+    margin-left: ${(prop: IMenuItemProps) => prop.selected ? wp('5%'): 0 };
+    margin-right: ${(prop: IMenuItemProps) => prop.selected ? wp('5%'): 0 };
+    background-color: ${(prop: IMenuItemProps) => prop.selected? COLORS.lunesBlack : COLORS.white };
+    border-color: ${(prop: IMenuItemProps) => prop.selected? COLORS.white : COLORS.lunesBlackUltralight };
```

Comment:
ternery does not really make sense here, as you put border-width to 0, so the color should not matter.